src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EgressOperationStatus.cs

```diff
@@ -20,6 +20,24 @@ public class OperationSummary
 
         [JsonPropertyName("status")]
         public OperationState Status { get; set; }
+
+        [JsonPropertyName("process")]
+        public OperationProcessInfo Process { get; set; }
+    }
+
+    /// <summary>
+    /// Represents the details of a given process used in an operation.
+    /// </summary>
+    public class OperationProcessInfo
+    {
+        [JsonPropertyName("pid")]
+        public int? ProcessId { get; set; }
```

Comment:
These properties should not be nullable. If the egress operation is missing process information, it's the EgressProcessInfo object that is null, not the individual properties on them.